src/containers/item-details/item-details.tsx

```diff
@@ -48,7 +49,7 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
                 likeCount={item.likes}
                 itemId={item.itemId.slice(0, 4)}
               />
-              {item.price && (
+              {item.price && item.status !== 0 ? (
```

Comment:
Please do not use hardcoded values. There is a ItemStatus enum available. If it happens to not exist, please create it and use it instead